ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -97,16 +98,16 @@ export default class UnlockPage extends Component {
         showOptInModal();
       }
     } catch ({ message }) {
+      this.failed_attempts += 1;
+
       if (message === 'Incorrect password') {
-        const newState = await forceUpdateMetamaskState();
+        // const newState = await forceUpdateMetamaskState();
```

Comment:
if you no longer need the newState, uncomment this and remove the `const newState =` -- we need to await the update but we don't need the resulting value